### ISSUE TITLE
Query Does Kestrel support OCSP Stapling

### ISSUE THREAD

**Issue by Tratcher** [aspnet/KestrelHttpServer#2244]
_From @rsrinivasanhome on January 5, 2018 13:28_

Does Kestrel Support OCSP Stapling ?  I am not able to find any links on google.

_Copied from original issue: aspnet/AspNetCoreModule#283_

**Comment by samsosa** [comment]
No, Kestrel doesn't support OCSP Stapling but Microsoft IIS since version 8.0 use this configuration by default.

**Comment by muratg** [comment]
After chatting with @shirhatti, putting this tentatively in 3.0.0.

**Comment by muratg** [comment]
cc @blowdart
